scripts/Dockerfile

```diff
@@ -10,6 +10,8 @@ ARG CONTROLLER_E2E_PATH=./${AWS_SERVICE}-controller/test/e2e
 # Path to the test-infra directory in the build context
 ARG TEST_INFRA_PATH=./test-infra
 
+ARG WEB_IDENTITY_TOKEN_FILE
```

Comment:
Please add both a default and a comment explaining exactly what this arg does. Also, if this is a target file path for where to copy the token file contents *to*, consider renaming the variable to `WEB_IDENTITY_TOKEN_DEST_PATH`